CONTRIBUTING.rst

```diff
@@ -1,36 +1,56 @@
-Contributing
-============
-
 Instructions for contributors
 -----------------------------
 
+In order to make a clone of the `GitHub <https://github.com/aio-libs/aiohttp>`_ repo: open the link and press the "Fork" button on the upper-right menu of the web page.
 
-In order to make a clone of the GitHub_ repo: open the link and press the
-"Fork" button on the upper-right menu of the web page.
-
-I hope everybody knows how to work with git and github nowadays :)
+If you'd like to learn more about Git and GitHub, `check out GitHub's helpful introduction
+<https://docs.github.com/en/get-started/using-git/about-git>`_.
 
 Workflow is pretty straightforward:
 
-  1. Clone the GitHub_ repo using the ``--recurse-submodules`` argument
+  0. Make sure you are reading the latest version of this document.
+     It can be found in the GitHub_ repo in the ``docs`` subdirectory.
+
+  1. Clone your forked GitHub_ repo with the ``--recurse-submodules`` flag as shown in the command below,
+     ensuring to replace the placeholder with your github username:
+
+      .. code-block:: shell
+
+         $ git clone \
+            https://github.com/<your_github_username>/aiohttp.git \
+            --recurse-submodules
+
 
   2. Setup your machine with the required development environment
 
   3. Make a change
 
   4. Make sure all tests passed
 
-  5. Add a file into the ``CHANGES`` folder, named after the ticket or PR number
+  5. Add a file into the ``CHANGES`` folder (see `Changelog update <CHANGES>`_ for how).
```

Comment:
This relative file reference only works on GitHub and is a broken link in Sphinx: https://aiohttp--9742.org.readthedocs.build/en/9742/contributing.html#instructions-for-contributors